packages/tsconfig.base.json

```diff
@@ -25,6 +25,7 @@
   "exclude": [
     "**/**.test.tsx",
     "**/**.test.ts",
-    "**/examples/**"
+    "**/examples/**",
+    "**/demos/Tabs/**"
```

Comment:
What purpose does adding this line serve? What error do you see if you didn't add it?

Reply:
oh yeah, nothing now... removing... I moved the TS file to examples and forgot about this